imsim/photon_ops.py

```diff
@@ -277,16 +272,15 @@ def get_camera_cached(camera_name: str):
 @photon_op_type("lsst_diffraction")
 def deserialize_lsst_diffraction(config, base, _logger):
     kwargs = config_kwargs(config, base, LsstDiffraction)
-    shift_optics = kwargs.pop("shift_optics", base.get("shift_optics", None))
 
     opsim_meta = galsim.config.GetInputObj(
         "opsim_meta_dict", config, base, "opsim_meta_dict"
     )
 
+    telescope = galsim.config.GetInputObj("telescope", config, base, "telescope")
```

Comment:
This should really be mentioned in the Register command that this input type is associated with this PhotonOp.  cf. http://galsim-developers.github.io/GalSim/_build/html/config_stamp.html#galsim.config.RegisterPhotonOpType